Jenkinsfile

```diff
@@ -104,6 +104,53 @@ pipeline {
                 }
             }
         }
+        stage("Clang-format") {
+            agent any
+            steps {
+                sh "printenv"
+                deleteDir()
+                retry(3) { checkout scm }
+                withCredentials([usernamePassword(credentialsId: 'a630aebc-6861-4e69-b497-fd7f496ec46b',
+                    usernameVariable: 'GIT_USERNAME', passwordVariable: 'GIT_PASSWORD')]) {
+                    sh """#!/bin/bash
+                        set -x
+                        git checkout -b ${branchName()}
```

Comment:
` ${branchName()}` should be `${GIT_BRANCH}` if you're looking for the current one checked out by Jenkins.